src/ray/protobuf/common.proto

```diff
@@ -456,3 +443,20 @@ enum WorkerExitType {
   // Worker exit due to placement group removal.
   PLACEMENT_GROUP_REMOVED = 3;
 }
+
+// The following enum should never be modified to maintain
+// backward compatibility in autoscaler.
+
+// Type of placement group strategy.
+enum PlacementStrategy { // Do not change!
```

Comment:
Please remove all instances of "Do not change"

Reply:
I added it to make sure the reviewer would see it, I was not sure if putting it on the top would be sufficient for the reviewer to notice.